test.php

```diff
@@ -13,7 +13,7 @@
 $WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' ) ? : '-o StrictHostKeyChecking=no';
 $WPT_PHP_EXECUTABLE = getenv( 'WPT_PHP_EXECUTABLE' ) ? : 'php';
-$WPT_PHPUNIT_CMD = getenv( 'WPT_PHPUNIT_CMD' ) ? : 'cd ' . escapeshellarg( $WPT_TEST_DIR ) . '; ' . $WPT_PHP_EXECUTABLE . ' phpunit.phar';
+$WPT_PHPUNIT_CMD = getenv( 'WPT_PHPUNIT_CMD' ) ? : 'cd ' . escapeshellarg( $WPT_TEST_DIR ) . ' && ./vendor/phpunit/phpunit/phpunit';
```

Comment:
Here, I noticed that we weren't using &&, so the return code of PHPUnit wouldn't be passed back.

Should I change it now, or keep that for a separate change/PR?

Reply:
Please always just force push on the existing PR so that it can be reviewed/tested at any given time and nobody will waste time on testing old versions.

Reply:
I’m sorry, I’m not sure what you are referring to.

There haven’t been any changes to this PR since you reported testing, and this thread/comment is about splitting out an extra change.

If you have an opinion on that, it’d be welcome, though!

Reply:
I've left this change in for now. If anyone has feedback on it, though, please feel free to open this discussion again.